webpack/scenes/ContentViews/Details/Versions/ContentViewVersions.js

```diff
@@ -157,7 +157,7 @@ const ContentViewVersions = ({ cvId, details }) => {
 
     return [
       <Checkbox
-        ouiaId={`select-version-${versionId}`}
+        ouiaId={`select-version-${rowIndex}`}
```

Comment:
Now that it's rowIndex, `select-version-x` implies that that's the name of the version. Maybe 
```suggestion
        ouiaId={`select-version-row-${rowIndex}`}
```
(same comment for all)

Reply:
I have made the suggested change